Std/Data/BinomialHeap/Basic.lean

```diff
@@ -502,6 +502,7 @@ variable {α : Type u} {le : α → α → Bool}
 /-- `O(1)`. Make a new empty binomial heap. -/
 @[inline] def empty : BinomialHeap α le := mkBinomialHeap α le
 
+instance : EmptyCollection (BinomialHeap α le) := ⟨.empty⟩
 instance : Inhabited (BinomialHeap α le) := ⟨.empty⟩
 
```

Comment:
Would it make sense to have an `Inhabited` instance derived from `EmptyCollection` higher up? It could always be overridden in the extremely unusual case where that's not the right choice for `default`.

Reply:
It does seem reasonable, but not here. Please either open an issue or a PR if you're interested in doing either!